src/bitbucket/BitbucketClient.ts

```diff
@@ -90,16 +90,16 @@ export class BitbucketClient {
     };
   }
 
-  createLandBuild(requestId: string, commit: string, variables: PipelinesVariables) {
-    return this.pipelines.createLandBuild(requestId, commit, variables);
+  createLandBuild(requestId: string, commit: string, variables: PipelinesVariables, lockId: Date) {
```

Comment:
jfmi why `lockId` is required for create but not for stop?

Reply:
and what will be the purpose of it?

Reply:
`lockId` is used in every function that is called from `next`. Since `next` transitions a request from queued to running it creates pipelines but doesn't stop them. I explain the purpose of the `lockId` at the bottom of the investigation page, but essentially it is to see what `next` call the functions are coming from.